### PR TITLE
Social Media Icons added

### PR DESCRIPTION
**Pull request by iArchitSharma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,20 @@
 - [Instagram](https://www.instagram.com/kunalsig/)
 
 ### Follow Community Classroom
-- [Twitter](https://twitter.com/commclassroom)
-- [LinkedIn](https://www.linkedin.com/company/commclassroom)
-- [Instagram](https://www.instagram.com/commclassroom)
+
+ 
+  <a href="https://discord.gg/K9kxUXvfND">
```
Comment (kaiwalyakoparkar) [inline README.md:17]: Change discord url to `https://discord.io/commclassroom`

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iArchitSharma) [reply README.md:17]: ok let me

Comment (iArchitSharma) [reply README.md]: Done

**Comment by kunal-kushwaha** [comment]
Thank You

**Pull request merged**